certbot-apache/certbot_apache/parser.py

```diff
@@ -60,10 +60,6 @@ def __init__(self, aug, root, vhostroot, version=(2, 4)):
         # Set up rest of locations
         self.loc.update(self._set_locations())
 
-        # Must also attempt to parse virtual host root
-        self._parse_file(self.vhostroot + "/" +
-                         constants.os_constant("vhost_files"))
```

Comment:
When we talked about this on the call I think we expressed some concerns about whether or not we can safely do this. In the conventional case, this works fine but what about people who have been manually setting `--apache-vhost-root`? These people may not exist, but this might break things for them correct?